purchase.adoc

```diff
@@ -1,675 +1,885 @@
-## Purchase API
+= Purchase API Reference Guide
+:toc: right
+:imagesdir: assets/images
 
-### Service Description
-The purchase API provides information about purchases made through iZettle.
+== About Purchase API
+Provides read only information about purchases made through Zettle.
+Purchase represents a receipt, it's immutable and contains information about items that were sold, one or several payments received, taxes, discounts etc.
+Refunds are represented by the same model but with negative amounts and refund related flags/fields.
 
-### URL
+=== Base URL
 https://purchase.izettle.com
 
-### Scopes
-The Purchase API implements the following scopes:
-
+=== OAuth Scope
 - `READ:PURCHASE`
 
-=== Get purchase history
-Will include cash register information if applicable. Purchases up to three years old are available to fetch.
+See https://github.com/iZettle/api-documentation/blob/master/authorization.adoc[OAuth2 API] for more information about access tokens.
+
+== __Fetch a list of purchases__
+__Returns a list of purchases for a merchant. Purchases up to three years old are available to fetch.__
+
+__GET  {nbsp}/purchases/v2__
+
+See <<example1>>.
```

Comment:
Example 2 and 3 seems to be not referenced. Do we need to add them or sould we remove them

Reply:
We leave them.